### PR TITLE
fix String.prototype.replaceAll

### PR DESCRIPTION
**Pull request by romainmenke**

### CODE REVIEW

File: polyfills/String/prototype/replaceAll/polyfill.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ CreateMethodProperty(String.prototype, 'replaceAll', function replaceAll(searchV
 			// 2.b.ii. Perform ? RequireObjectCoercible(flags).
 			RequireObjectCoercible(flags)
 			// 2.b.iii. If ? ToString(flags) does not contain "g", throw a TypeError exception.
-			if (ToString(flags).includes('g')) {
+			if (!ToString(flags).includes('g')) {
```
Comment (JakeChampion) [inline polyfills/String/prototype/replaceAll/polyfill.js:19]: Oh my!

File: polyfills/_ESAbstract/GetSubstitution/polyfill.js [hunk 2]
```diff
@@ -1,73 +1,72 @@
...
-					i += 1;
-				} else if (next === '&') {
-					result += matched;
-					i += 1;
-				} else if (next === '`') {
-					result += position === 0 ? '' : str.slice(0, position - 1);
-					i += 1;
-				} else if (next === "'") {
-					result += tailPos >= stringLength ? '' : str.slice(tailPos);
+function isDigit(string) {
```
Comment (JakeChampion) [inline polyfills/_ESAbstract/GetSubstitution/polyfill.js:3]: This was in an IIFE to ensure that isDigit is not leaked to the outer scope of the polyfill bundle, I think we need to keep the IIFE here

Comment (romainmenke) [reply polyfills/_ESAbstract/GetSubstitution/polyfill.js:3]: Ok!

**Comment by romainmenke** [comment]
note : fails in IE

**Comment by romainmenke** [comment]
This needs a polyfill for `RegExp.prototype[@@replace]()`

**Comment by romainmenke** [comment]
@JakeChampion going to close this one.
Started from scratch.

**Pull request closed without merging**